<launch>
  <node name="imlidar_node" pkg="imlidar_driver" type="imlidar_node" output="screen" >
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud_rate" value="115200" />
    <param name="frame_id" value="imlidar" />
    <param name="rps" value="8" />
    <param name="data_sequence_direction" value="cw" />
  </node>
</launch>
